website/content/docs/k8s/operations/server-certificate-rotation.mdx

```diff
@@ -0,0 +1,27 @@
+---
+layout: docs
+page_title: Server Certificate Rotation
+sidebar_title: Server Certificate Rotation
+description: Rotate Server Certificate on Kubernetes Cluster safely
+---
+
+# Rotating Server Certificates
+
+As of Consul Helm version `0.29.0`, if TLS is enabled, new TLS certificates for the Consul Server
+are issued every time the TLS-init job runs. These certificates are signed by the same CA and will
+continue to work as expected in the existing cluster.
+
+Consul serves read the certificates during start-up and keep them in-memory. In order to ensure the
+servers utilize the newer certificate, the servers pods need to be [restarted safely](/docs/k8s/operations/upgrade#upgrading-consul-servers).
```

Comment:
```suggestion
servers use the newer certificate, the server pods need to be [restarted safely](/docs/k8s/operations/upgrade#upgrading-consul-servers).
```

Reply:
I think we should say that in some cases a helm upgrade will not cause an upgrade of the consul servers, and in that case the servers will not pick up the new certificates.